### ISSUE TITLE
ChangeLog in Update Message

### ISSUE THREAD

**Issue by MainConMan** [absolllute/Mega-Hack-Pro-Future#448]
When you updated MHv7 and open the game, a message will pop up saying that auto-update was successful, and tell you what version you're on. It would be cool if there was a change log in that message too.

**Comment by Nicko2102** [comment]
> When you updated MHv7 and open the game, a message will pop up saying that auto-update was successful, and tell you what version you're on. It would be cool if there was a change log in that message too.

I like this, either the actual list or a link to the changelog on absolute's website or wherever it ends up,  or a combination (list a few major ones, and provide a link if people want to see the rest)

**Comment by MainConMan** [comment]
> > When you updated MHv7 and open the game, a message will pop up saying that auto-update was successful, and tell you what version you're on. It would be cool if there was a change log in that message too.
> 
> I like this, either the actual list or a link to the changelog on absolute's website or wherever it ends up,  or a combination (list a few major ones, and provide a link if people want to see the rest)

I usually check Absolute's twitter for the change log, since the website doesn't show beta change logs. So the betas would have to link to that. But otherwise, I think this is also a good idea!

**Comment by Nicko2102** [comment]
Quick suggestion - title looks like "Change Login Update message", would suggest an edit so it reads Changelog in update message. Also I'll contact you on discord after my class and we can get to work on the documentation :D

**Comment by NewAlexGamer** [comment]
maybe make it so there's a specific button that you can press under the "mega hack pro" label whenever you want to see the version's change log instead of opening another program (browser in this case) every time or just seeing it once every update, which occures not very often

**Comment by Brittank88** [comment]
> It would be cool if there was a change log in that message too.

Can automatically pull that from a changelog that is embedded in the corresponding milestone's description maybe?

Or maybe from a `CHANGELOG.md` file on this repo (which could be cleaner)?

**Comment by absoIute** [comment]
I would probably just link the changelog instead ngl, but yes!

**Comment by Brittank88** [comment]
Dropping this here in case anyone finds it interesting / it turns out to be useful: https://keepachangelog.com/en/1.0.0/

**Comment by Brittank88** [comment]
> I would probably just link the changelog instead ngl

Yeah true, it's better than maintaining another markdown file. At least the All-Contributors bot automatically manages `CONTRIBUTORS.md`.

**Comment by Brittank88** [comment]
@all-contributors please add @absoIute for bug, business, code, content, doc, design, financial, fundingFinding, ideas, infra, maintenance, projectManagement, question and security.

**Comment by Brittank88** [comment]
Now that I've configured the bot properly, let's try this again.

https://github.com/all-contributors please add @absoIute for bug, business, code, content, doc, design, financial, fundingFinding, ideas, infra, maintenance, projectManagement, question and security.

**Comment by Brittank88** [comment]
@All-Contributors please add @absoIute for bug, business, code, content, doc, design, financial, fundingFinding, ideas, infra, maintenance, projectManagement, question and security.

**Comment by Brittank88** [comment]
@All-Contributors please add @absoIute for bug, business, code, content, doc, design, financial, fundingFinding, ideas, infra, maintenance, projectManagement, question and security.

**Comment by Brittank88** [comment]
This is not working properly but I'll stop bloating this thread with it and create a new dummy issue where I can do my own thing.
